### ISSUE TITLE
Documentation: wording of the definition of a jump process 

### ISSUE THREAD

**Issue by hurak** [SciML/JumpProcesses.jl#445]
The mathematical overview in the documentation at 
https://github.com/SciML/JumpProcesses.jl/blame/9e272f797d1dfd23f79cc61a737d2a296e5f699d/docs/src/index.md#L52-L53C33 essentially reads that the random process dN(.) attains at a particular time t a value 1 with some probability. Am I interpreting it correctly? I would argue that this might misleading since such probability is always zero for a process defined over a continuous time, isn't it? Perhaps some rewording could help. I am not fluent with the stochastic process lingo, though.

**Comment by isaacsas** [comment]
Yes, that could use some rewriting for clarity. Thanks for pointing it out.
